### DISCUSSION TITLE
多线情况下PaoPaoDNS如何使用

### DISCUSSION THREAD

**Discussion by tsreal** [kkkgo/PaoPaoDNS#167] [问答和帮助] [answered]
家中因为各种原因临时有三条可用宽带（移动联通电信各一条），拿ikuai接入汇聚和分流使用，此时ikuai接口为wan1，wan2，wan3，lan1。上网通过匹配规则来选择不同运营商出口传输数据。请问这种情况下需要为paopaodns固定一个运营商线路查询（ikuai端口分流）以获得更稳定的解析结果吗？
烦请解惑，感谢🙇‍

**Comment by kkkgo** [comment, marked as answer]
主要看你想实现怎么样的分流。如果你想主要使用电信，那么固定为电信出口是比较稳妥的做法。如果你不在乎主要用哪条线路，那么解析结果也自然不需要固定是哪个运营商。
